sql/updates/latest-dev.sql

```diff
@@ -46,3 +46,6 @@ BEGIN
     WHERE mat_hypertable_id OPERATOR(pg_catalog.=) _cagg_data.mat_hypertable_id;
 END;
 $BODY$;
+
+ALTER TABLE _timescaledb_catalog.dimension ADD COLUMN compress_interval_length bigint NULL;
+ALTER TABLE _timescaledb_catalog.dimension ADD CONSTRAINT dimension_compress_interval_length_check CHECK (compress_interval_length IS NULL OR compress_interval_length > 0);
```

Comment:
Doing catalog table changes like that is problematic and will lead to different schema in new installations vs upgraded installation. You need to rebuild any catalog table you intend to update to prevent that. There are plenty of examples in previous update scripts.